config/config.php

```diff
@@ -112,6 +113,12 @@
             'guzzle' => [],
         ],
 
+        'linkedin-openid' => [
+            'client_id' => env('LINKEDIN_OPENID_CLIENT_ID'),
+            'client_secret' => env('LINKEDIN_OPENID_CLIENT_SECRET'),
+            'guzzle' => [],
+        ],
```

Comment:
@mjauvin what's the difference between the two providers?

Reply:
The main difference is the scopes they pass to the oauth authentication page:
- The `linkedin` provider requires those scopes: `['r_liteprofile', 'r_emailaddress']`
- The `linkedin-openid` provider requires those scopes: `['openid', 'profile', 'email']`

So, depending on the products you choose for your LinkedIn App, it's harder to get the scopes (permissions) required by the regular `linkedin` provider than it is with the `linkedin-openid` one because there's a `Sign In with LinkedIn using OpenID Connect` product that adds exactly those permissions to your app.

Reply:
Linking to https://github.com/laravel/socialite/issues/695 as it provides more background, basically `linkedin` is deprecated and impossible to get a new app issued the appropriate scopes. Only `linkedin-openid` is allowed for new apps going forward.

Any thoughts on just removing the `linkedin` provider from our default config? Also is LinkedIn supported without any other packages or does it require an additional package to be installed?

Reply:
> Linking to [laravel/socialite#695](https://github.com/laravel/socialite/issues/695) as it provides more background, basically `linkedin` is deprecated and impossible to get a new app issued the appropriate scopes. Only `linkedin-openid` is allowed for new apps going forward.

Makes sense.

> Any thoughts on just removing the `linkedin` provider from our default config? Also is LinkedIn supported without any other packages or does it require an additional package to be installed?

No objection to remove `linkedin` provider. No other dependencies required.

Reply:
It's been removed.